bentoml/_internal/cli/containerize.py

```diff
@@ -147,6 +150,7 @@ def add_containerize_command(cli: click.Group) -> None:
     @click.option(
         "--ulimit", type=click.STRING, default=None, help="Ulimit options (default [])."
     )
+    @kwargs_transformers(transformer=lambda x: None if not x or (isinstance(x, tuple) and not x) else x)  # type: ignore
```

Comment:
What's this if expression trying to do?

Reply:
tries to transformers all kwargs to None if it is "" or ("",)